src/vivarium_public_health/metrics/mortality.py

```diff
@@ -78,14 +78,18 @@ def setup(self, builder: Builder) -> None:
         self._cause_components = builder.components.get_components_by_type(
             (DiseaseState, RiskAttributableDisease)
         )
-
+        self.causes_of_death = ["other_causes"] + [
```

Comment:
These 3 attributes are not functionality changes they will just make subclassing easier.